hasura.planx.uk/tests/lowcal_sessions.test.js

```diff
@@ -428,4 +428,19 @@ describe("lowcal_sessions", () => {
       });
     });
   });
+
+  describe("platformAdmin", () => {
+    let i;
+    beforeAll(async () => {
+      i = await introspectAs("platformAdmin");
+    });
+
+    test("cannot query lowcal_sessions", () => {
+      expect(i.queries).not.toContain("lowcal_sessions");
+    });
+
+    test("cannot create, update, or delete lowcal_sessions", () => {
+      expect(i).toHaveNoMutationsFor("lowcal_sessions");
+    });
+  });
```

Comment:
Updated introspection tests are very helpful to quickly understand intended access-level, thanks !  

One clarification though, when I sign into this pizza's Hasura console with my user, who is _not_ a Platform Admin according to the `teams` table, I can indeed query tables like `lowcal_sessions` - is that expected? I guess so because of these headers, which sort of still inherently contradict value of `teams.is_platform_admin` ?? 
```
"x-hasura-allowed-roles": ["platformAdmin"],
"x-hasura-default-role": "platformAdmin",
```

Reply:
This is a good question.

When we log into the console, we always use the `admin` role - you're never logged in as your user and have "god mode".

However, using the GraphiQL playground in Hasura you can uncheck the `x-hasura-admin-secret` header, and add a `x-hasura-role` header (set to platformAdmin) and then queries and mutations will reflect this permission level.

The follow up step to this PR would be to make sure we always pass the correct headers to Hasura (it's hardcoded to the admin secret in `planx-core` currently). Requests made via Apollo in the frontend already correctly pass your JWT through to Hasura.

Reply:
Makes sense now, thanks !